### PR TITLE
Add test for yieldlab targeting fix

### PR DESCRIPTION
**Pull request by muuki88**
Add regression tests for #1131

### CODE REVIEW

File: src/test/java/org/prebid/server/bidder/yieldlab/YieldlabBidderTest.java [hunk 1]
```diff
@@ -111,6 +111,59 @@ public void makeHttpRequestsShouldSendRequestToModifiedUrlWithHeaders() {
...
+                .device(Device.builder().ip("ip").ua("Agent").language("fr").devicetype(1).build())
+                .regs(Regs.of(1, ExtRegs.of(1, "usPrivacy")))
+                .user(User.builder().buyeruid("buyeruid").ext(ExtUser.builder().consent("consent").build()).build())
+                .site(Site.builder().page("http://www.example.com").build())
+                .build();
+
+        // when
+        final Result<List<HttpRequest<Void>>> result = yieldlabBidder.makeHttpRequests(bidRequest);
+
+
```
Comment (rpanchyk) [inline src/test/java/org/prebid/server/bidder/yieldlab/YieldlabBidderTest.java:154]: Our codestyle plugin is very angry ;)
`There is more than 1 empty line one after another. [EmptyLineSeparator]`

Comment (muuki88) [inline src/test/java/org/prebid/server/bidder/yieldlab/YieldlabBidderTest.java:154]:
```suggestion

```

Comment (muuki88) [reply src/test/java/org/prebid/server/bidder/yieldlab/YieldlabBidderTest.java]: Let's give him what he wants :joy:

**Comment by muuki88** [comment]
Oh wow! How does this work? The build system uses java 8, but target is set to java 8?

**Comment by rpanchyk** [comment]
We use Java 8 for both:
```
    <maven.compiler.source>1.8</maven.compiler.source>
    <maven.compiler.target>1.8</maven.compiler.target>
```

**Comment by muuki88** [comment]
My bad :grimacing: I haven't read a `pom.xml` in years and just saw the java11 profile in the `profiles` section.

**Review by rpanchyk — APPROVED** [review]

**Pull request merged**